File: Client/java/apisdk/models/Category.java
package apisdk.models;

import com.microsoft.kiota.serialization.ValuedEnum;
import java.util.Objects;

@jakarta.annotation.Generated("com.microsoft.kiota")
public enum Category implements ValuedEnum {
    HelpDocs("helpDocs"),
    PublicHubs("publicHubs"),
    TrackDb("trackDb");
    public final String value;
    Category(final String value) {
        this.value = value;
    }
    @jakarta.annotation.Nonnull
    public String getValue() { return this.value; }
    @jakarta.annotation.Nullable
    public static Category forValue(@jakarta.annotation.Nonnull final String searchValue) {
        Objects.requireNonNull(searchValue);
        switch(searchValue) {
            case "helpDocs": return HelpDocs;
            case "publicHubs": return PublicHubs;
            case "trackDb": return TrackDb;
            default: return null;
        }
    }
}

File: Client/php/Models/Category.php
<?php

namespace ApiSdk\Models;

use Microsoft\Kiota\Abstractions\Enum;

class Category extends Enum {
    public const HELP_DOCS = 'helpDocs';
    public const PUBLIC_HUBS = 'publicHubs';
    public const TRACK_DB = 'trackDb';
}
